### PR TITLE
Trying to get some docs up on how to run things on AWS

### PR DESCRIPTION
**Pull request by jwills**
Creating this as a draft so I can get some feedback while I work through the rest of the instructions.

### CODE REVIEW

**Comment by samshah** [comment]
thank you for writing this up! FYI: there's a [CSP wiki](https://github.com/HopkinsIDD/COVID19_Minimal/wiki/Getting-Started), not sure where these docs should go.

File: runbook/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+has become, an `r5.8xlarge` should be perfectly adequate.)
+
+If you have access to the `jh-covid` account, you should use the *IDD Staging AMI* (`ami-03641dd0c8554e5d0`) to provision and launch new
+staging servers; it is already setup with all of the dependencies described in this section. You can find the AMI [here](https://us-west-2.console.aws.amazon.com/ec2/v2/home?region=us-west-2#Images:sort=imageId), select it, and press the *Launch* button to walk you through the Launch Wizard to choose your instance type and `.pem` file to provision your staging server.
+Once your instance is provisioned, be sure to put its IP address into the `HostName` section of the `~/.ssh/config` file on your local client so that you can connect to it from your
+client by simply typing `ssh staging` in your terminal window.
+
+If you do *not* have access to the `jh-covid` account, you should walk through the regular EC2 Launch Wizard flow and be sure to choose the
+*Amazon Linux 2 AMI (HVM), SSD Volume Type* (`ami-0e34e7b9ca0ace12d`, the 64-bit x86 version) AMI. Once the machine is up and running and you can
```
Comment (samshah) [inline runbook/README.md:54]: amazon linux vs ubuntu, latter of which we're running in the container?

**Comment by jwills** [comment]
> thank you for writing this up! FYI: there's a [CSP wiki](https://github.com/HopkinsIDD/COVID19_Minimal/wiki/Getting-Started), not sure where these docs should go.

I am more-or-less incapable of writing anything in a browser b/c Twitter will distract me, so I do everything in vim. But happy to migrate the text to the wiki once I finish the draft if that is the preferred venue!

File: runbook/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+SSH to it, you will need to run the following code to install the software you will need for the rest of the run:
+
+```
+sudo yum -y update
+sudo yum -y install awscli 
+sudo yum -y install git 
+sudo yum -y install docker.io 
+sudo yum -y install pbzip2 
+
```
Comment (samshah) [inline runbook/README.md:63]: if you add `sudo usermod -G docker` to add the current user to the docker group, you won't have to sudo docker

Comment (jwills) [reply runbook/README.md:63]: Ooh, TIL!

File: runbook/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+For this example, I'm going to use the `COVID19_Minimal` repo as my example, so I would run `git clone git@github.com:HopkinsIDD/COVID19_Minimal.git`
+to get it onto the staging server. By convention, I usually do runs (for both Planning and Inference jobs) with the `COVIDScenarioPipeline` repository
+nested inside of the data repository, so I would then do `cd COVID19_Minimal; git clone git@github.com:HopkinsIDD/COVIDScenarioPipeline.git` to
+clone the modeling code itself into a child directory of the data repository.
+
+## Getting and Launching the Docker Container
+
+The previous section is only for getting a minimal set of dependencies setup on your staging server. To do an actual run, you will
+need to download the Docker container that contains the more extensive set of dependencies we need for running the code in the
+`COVIDScenarioPipeline` repository. For *Inference* jobs, please run `sudo docker pull hopkinsidd/covidscenariopipeline:latest-dev`;
```
Comment (samshah) [inline runbook/README.md:94]: i always use the dev container for both planning and inference

Comment (jwills) [reply runbook/README.md:94]: That would certainly simplify things. 🤔

**Comment by jwills** [comment]
Now wiki-fied: https://github.com/HopkinsIDD/COVIDScenarioPipeline/wiki/AWS-Instructions

**Comment by eclee25** [comment]
No longer needed

**Pull request closed without merging**